devices/Sps30/packages.config

```diff
@@ -3,7 +3,7 @@
   <package id="nanoFramework.CoreLibrary" version="1.12.0" targetFramework="netnanoframework10" />
   <package id="nanoFramework.Runtime.Events" version="1.10.0" targetFramework="netnanoframework10" />
   <package id="nanoFramework.Runtime.Native" version="1.5.4" targetFramework="netnanoframework10" />
-  <package id="nanoFramework.System.Buffers.Binary.BinaryPrimitives" version="1.1.113.2032" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.System.Buffers.Binary.BinaryPrimitives" version="1.1.113.2032" targetFramework="netnano1.0" />
```

Comment:
why adjusting this file?

Reply:
I had a lot of issues with nuget packages not restoring and references not being resolved. I tried a couple of things, including this.

Reply:
ok but can we exclude this file from this PR? So the PR only concerns this new SDS30?

Reply:
Whoops! I did not notice the Sps30 there! I've rebased and edited the commit to leave that one out.